peachjam/adapters/gazettes.py

```diff
@@ -78,8 +78,12 @@ def update_document(self, expression_frbr_uri):
             file_path = ga_source_file[0].pop("file")
 
             if ga_source_file:
+                source_url = (
+                    f"https://archive.gazettes.africa/{file_path.split(':')[2]}"

```

Comment:
I think it's better to use `https://gazettes.africa/akn/za/officialGazette/government-gazette/2022-01-03/45717/eng@2022-01-03/source` because:

1. the archive path is internal to how gazettes.africa works
2. gazettes.africa will redirect the `.../source` URL to the correct place
3. we don't have to guess the archive path URL, which is internal to gazettes.africa
4. we know exactly how to build the `.../source` URL with the FRBR URI of the gazette.

Reply:
I've changed it to use gazettes.africa/source.